container_files/tutorials/Identifying biased features.ipynb

```diff
@@ -0,0 +1,1689 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Identifying biased features\n",
```

Comment:
Il manque peut-être 2-3 séparations dans le notebook. Style:

a) Importing the data
b) Training a first model
c) Retraining without the biased feature
d) Conclusion (qui manque). qqchose genre la feature la plus haute n'est pas toujours problématique, et une bonne compréhension de son dataset est importante. Ça donne par contre une très bonne indication et est un outil essentiel dans le toolbox d'un data scientist.